src/items/itemDialog.component.tsx

```diff
@@ -1146,6 +1147,11 @@ function ItemDialog(props: ItemDialogProps) {
             }
             onClick={type === 'edit' ? handleEditItem : handleAddItem}
             sx={{ mr: 3 }}
+            endIcon={
+              isAddItemsPending || isAddItemPending || isEditItemPending ? (
+                <CircularProgress size={16} />
```

Comment:
```suggestion
                <CircularProgress size={20} />
```

Reply:
The dialogs that are steppers have a smaller size as the actual button is smaller, so it fits proportionally better with the `Finish` button